Explain form's reorderable $condition parameter (#9059)
laravel-filament/filament: packages/tables/docs/10-advanced.md

}
```

> The `reorderable()` method also accepts, as a second parameter, a boolean condition to control whether the reorderable feature will be enabled:

Reviewer: ```suggestion
The `reorderable()` method also accepts a boolean condition as its second parameter, allowing you to conditionally enable reordering:
```
